src/main/java/frc/robot/commands/vision/Align.java

```diff
@@ -0,0 +1,95 @@
+package frc.robot.commands.vision;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+import org.photonvision.EstimatedRobotPose;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.util.Vision;
+
+public class Align extends CommandBase{
```

Comment:
I think align is pretty vauge and a better name such as "apriltagAngleAlignTest" would be more discriptive